carbon-design-system/carbon
# fix(notification): actionable non-inline content close button layout bug

Fixes `ActionableNotification` - content text was running into the close button due to an improperly scoped style.

#### Changelog

**Changed**

- ActionableNotification: text content wraps to respect close button layout

#### Testing / Reviewing
- View the ActionableNotification story in the carbon-next storybook. 
- Put some text into the children prop control
- Validate text doesn't wrap into the close button
- Make `inline` prop `true`, and do the same test to ensure no regressions for inline variant.

<details>
  <summary>Screenshots</summary>

  | bug 🐞 | fixed ✅ |
| - | - |
| ![Screen Shot 2021-09-29 at 3 07 01 PM](https://user-images.githubusercontent.com/3360588/135343042-2aa60bc2-fb95-410b-81eb-75d9ff4e8f8b.png) | ![Screen Shot 2021-09-29 at 3 20 37 PM](https://user-images.githubusercontent.com/3360588/135343058-9cc5a54d-9cf7-499a-b3ae-af327403397f.png) |
</details>
